doc/content/gettingStarted/start.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Configure & Start"
+description: ""
+weight: 2
+draft: false
+--- 
+
+## <a name="running">Running the stack</a>
+
+Download our [Docker Compose configuration](../docker-compose.yml) example. The stack can be run without any configuration, but for the purpose of this guide
+we provided you a basic one.
+
+With the `docker-compose.yml` file in the directory of your terminal prompt, enter the following commands to initialize the database, create the first user `admin`, create the CLI OAuth client and start the stack:
+
+* Download the necessary `images`:
+```bash
+$ docker-compose pull
+```
+{{< asciinema 6eF6gN7BMwETjwY3ABYEpuHWx >}}
```

Comment:
Is this some external reference? How does this work offline?

Reply:
Yes, https://asciinema.org/. It's free.
When it is offline it just won't show.